llvm/lib/Transforms/Vectorize/VectorCombine.cpp

```diff
@@ -4223,6 +4485,9 @@ bool VectorCombine::run() {
         if (foldCastFromReductions(I))
           return true;
         break;
+      case Instruction::ExtractElement:
+        if (foldShuffleChainsToReduce(I))
+          return true;
```

Comment:
Missing break.

Reply:
How did it pass before?

Reply:
Only some post-commit buildbots build llvm with `-Werror`.